src/shared/hooks-util.ts

```diff
@@ -1,4 +1,4 @@
-import type { HookCallback, HookEnvMap, Hooks } from '../core/hooks';
+import type { HookCallback, HookEnvMap, Hooks } from '../core/classes/hooks';
```

Comment:
Is `HooksEnvMap` something to be added later? I can't find this type anywhere.

I can't add a suggestion to line 17, so show it here for not to lose:

```diff
- export function addHooks (hooks: Hooks, hooksMap: { [K in keyof HookEnvMap]?: HookCallback<K> }) {
+ export function addHooks (hooks: Hooks, hooksMap: { [K in keyof HookEnvMap]?: HookCallback }) {
```

Reply:
I think it's part of the stuff that got removed.